mayastor/src/lvm/lvm.rs

```diff
@@ -0,0 +1,285 @@
+use tokio::process::Command;
+use crate::lvm::error::Error;
+
+use rpc::mayastor::{CreatePoolRequest, Pool, PoolType, PoolState};
+
+const PVCREATE_COMMAND: &str = "pvcreate";
+const PVS_COMMAND: &str = "pvs";
+const PVREMOVE_COMMAND: &str = "pvremove";
+const VGCREATE_COMMAND: &str = "vgcreate";
+const VGS_COMMAND: &str = "vgs";
+const VGREMOVE_COMMAND: &str = "vgremove";
+
+pub async fn lookup_lvm_vg(name: &str) -> Option<Pool> {
+    let pools = list_lvm_vg().await.ok()?;
+    match pools.iter().find(|p| p.name == name) {
+        Some(p) => {
+            return Some(Pool{
+                name: p.name.as_str().to_owned(),
+                disks: p.disks.as_slice().to_owned(),
+                capacity: p.capacity,
+                used: p.used,
+                pooltype: PoolType::Lvm.into(),
+                state: PoolState::PoolOnline.into(),
+            });
+        }
+        None => None
+    }
+}
+
```

Comment:
If I call recall directly, we were not planning to "create" pools rather only import them. I really do not want to get into the work related to created pools at this point. It's simple to create pools but even simpler to create them incorrectly when considering JBOD redundancy for example. We should document that this is purely for the simplest cases.

Reply:
Few doubts about pools:
- Will we only import existing `volumegroups` as mayastor pools? 
- Or will we allow the creation of `volumegroups` from mayastor as well by passing unused disks?
- Right now this code only supports creation of vgs and while creating we add the tag `mayastor` to identify it.
- For importing we can add the `mayastor`  tag to the existing volumegroups so that it will be identified as being in use by mayastor as pools. 
- Also for importing a volumegroup the disks argument is not required, how should we handle that case?

Reply:
Perhaps later we will add the ability to "create" VGs, but for now, we do not. If the disk argument is not needed simple mark the argument name as disks: _disk and it will indicate an unused variable.